packages/asset-registry/src/blockchain-facade/ConsumingAsset.ts

```diff
@@ -72,7 +72,7 @@ export const getAllAssetsOwnedBy = async (owner: string, configuration: Configur
 
 export class Entity extends Asset.Entity implements Asset.IOnChainProperties {
     getUrl(): string {
-        const consumingAssetLogicAddress = this.configuration.blockchainProperties.consumingAssetLogicInstance.web3Contract._address;
+        const consumingAssetLogicAddress = this.configuration.blockchainProperties.consumingAssetLogicInstance.web3Contract.options.address;
```

Comment:
Is this preferred way of accessing address?

Reply:
Yeah. That seems to be the only way if you follow the `Contract` type. which we now do.